pkg/auth/api/types.go

```diff
@@ -40,6 +40,7 @@ type Client interface {
 	GetSecret() string
 	GetRedirectUri() string
 	GetUserData() interface{}
```

Comment:
pretty sure this was made to match the osin client interface... we'll likely need to include the grant strategy in the "user data"... that's our bucket to plumb whatever data we want through all the osin apis